tools/filters/joiner.xml

```diff
@@ -85,7 +85,7 @@ ${json.dumps( __fill_options )}
    </param>
   </inputs>
   <outputs>
-     <data format_source="input1" name="out_file1" metadata_source="input1" />
+     <data auto_format="true" name="out_file1" />
```

Comment:
can you just make this `tabular` ? 
```suggestion
     <data format="tabular" name="out_file1" />
```

This would be correct in all instances, while joining subclasses of tabular, like gtf or gff are not valid gff or gtf.

Reply:
I was swinging between a fixed one, `tabular`, and auto. Other people also mentioned `tabular` in previous discussion. There will still be discrepancy existing though as uploaded dataset could be sniffed to other format, like `interval`, by autodetect. Anyway, I'm totally fine with `tabular`. @jennaj  what do you think?

Reply:
@mvdbeek  I made the change. There is another issue associated with this one. I'll have it in a try. Please keep this PR open. Thanks.